src/main/java/eu/europeana/metis/sandbox/service/workflow/EnrichmentServiceImpl.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.stereotype.Service;
 
 @Service
-class DefaultEnrichmentService implements EnrichmentService {
+class EnrichmentServiceImpl implements EnrichmentService {
```

Comment:
Just an observation here about those classes and their corresponding interfaces.
Do you think you need to have all those pairs of interfaces and implementation per step, cause they look very similar to each other, or just a general interface and all the implementation steps inheriting from that one would be enough?

Reply:
I think that is an option too, I kept them separate because each step by itself is different, and naming the method in the contract reflects more on what it does. But as the code is right now we might be able to go for that option if we decide. Can we revisit this item after having some more stories are done? to see how the application evolves

Reply:
Yes ofc